pipelines/tekton/aiedge-e2e/templates/build-container-image-buildah-cache-pvc.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: buildah-cache-pvc
+  labels:
+    app: TODO-INSERT-AIEDGE-PIPELINE-UNIQUE-APP-LABEL
```

Comment:
Maybe change this to `rhoai-edge-pipelines` like in the `kustomization.yaml` file?

Reply:
Done.  That was originally a placeholder reminder to add some commonLabel so that a reviewer would catch it if I forgot to update it